### DISCUSSION TITLE
Unable to update lazy.nvim, got error saying "doc/lazy.nvim.txt" is changed

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#786] [Q&A] [answered]
I got the following error when failed to update lazy.nvim: (images attached too)

```
  Failed (1)
    ● lazy.nvim 10.61ms   init.lua
        error: Your local changes to the following files would be overwritten by checkout:
        	doc/lazy.nvim.txt
        Please commit your changes or stash them before you switch branches.
        Aborting
```

I don't remember that I changed `doc/lazy.nvim.txt` file, and I'm unable to locate it in the home directory. 

My [nvim config](https://github.com/hopezh/OatVim) is synced with github and there is no uncommitted changes, as shown in the image below. 

May I ask how to solve this issue? 

According to [this post](https://github.com/LazyVim/LazyVim/discussions/368), shall I uninstall lazy.nvim itself? 

![image](https://github.com/LazyVim/LazyVim/assets/23435810/026e7202-79de-4ac3-b660-560b5aa1d4b6)

![image](https://github.com/LazyVim/LazyVim/assets/23435810/3cc5cbe8-eb5d-4eff-8927-bc65dc3a1946)

![image](https://github.com/LazyVim/LazyVim/assets/23435810/f77464cc-d565-4467-867f-8948eda23446)

**Comment by folke** [comment, marked as answer]
You did change it as the error says.

IT's locatied under `~/.local/share/lazy/lazy.nvim/doc...`

**Reply by hopezh** [reply]
Thanks. I didn't get this error/warning after I replaced my nvim config with a fresh clone from github. 

![image](https://github.com/LazyVim/LazyVim/assets/23435810/8321f61d-ac16-4430-b796-8ae0dc39392a)
